core/Engine.go

```diff
@@ -105,6 +105,9 @@ func (v *Engine) LoadFile(fileName string) []byte {
 	}
 	for _, mpqFile := range v.Settings.MpqLoadOrder {
 		archive, _ := mpq.Load(path.Join(v.Settings.MpqPath, mpqFile))
+		if archive == nil {
```

Comment:
Another option would be to check the returned error, instead of discarding it to `_`.

```go
archive, err := mpq.Load(...)
if err != nil {
   log.Fatalf("Failed to load specified MPQ file: %s; %v", mpqFile, err)
}
```